docs/csharp/language-reference/operators/default.md

```diff
@@ -5,6 +5,8 @@ ms.date: 03/13/2020
 f1_keywords:
   - "default_CSharpKeyword"
   - "default"
```

Comment:
We can delete this to avoid duplicate f1_keywords.
But **don't** delete it from `keywords/default.md` for backcompat with older Roslyn versions.

```suggestion
```

Reply:
I think this is a good change.

Reply:
As mentioned in the Roslyn PR, it also means one less added keyword, since we can now just use the existing `default_CSharpKeyword` one that's already on the operator page.